packages/editor/src/plugins/image/controls/settings-modal-controls.tsx

```diff
@@ -27,7 +34,15 @@ export function SettingsModalControls({ state }: Pick<ImageProps, 'state'>) {
         // eslint-disable-next-line @typescript-eslint/no-base-to-string
         value={isTemp ? '' : src.value.toString()}
         disabled={isTemp && !isFailed}
-        onChange={(e) => src.set(e.target.value)}
+        onChange={(e) => {
+          src.set(e.target.value)
+        }}
+      />
+      <LicenseDropdown
+        defaultLicense={licence?.defined ? licence.value : undefined}
+        onLicenseChange={updateOrCreateLicence}
+        // eslint-disable-next-line @typescript-eslint/no-base-to-string
+        isPixabayImage={src.get().toString().includes('Pixabay Lizenz')}
```

Comment:
When would the src ever include `'Pixabay Lizenz'`?

Reply:
Typo, thanks, fixed